cmd/root.go

```diff
@@ -31,13 +25,6 @@ func Execute() {
 }
 
 func init() {
-	// Here you will define your flags and configuration settings.
-	// Cobra supports persistent flags, which, if defined here,
-	// will be global for your application.
-
-	// rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.awsctl.yaml)")
-
-	// Cobra also supports local flags, which will only run
-	// when this action is called directly.
 	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
+	rootCmd.PersistentFlags().String("region", "", "--region=us-east-1")
```

Comment:
Write help message

Reply:
Something like ``awsctl get ec2 --region --help`` ??

Reply:
instead of `--region=us-east-1`, add something meaningful, maybe you can refer AWS CLI